web/src/app/lists/FlatList.js

```diff
@@ -46,8 +54,10 @@ export default class FlatList extends React.PureComponent {
           title: p.node.isRequired,
           subText: p.node,
           action: p.element,
+          button: p.element, // a secondary button element to be placed to the left of the action
```

Comment:
The first action is already a "secondaryAction" by material (as clicking the list item itself is the primary action), so I wanted to avoid any discrepancy there.

Reply:
`FlatList` has it's own API/props that should be consistent for the component as a whole, regardless of current implementation details under the covers. We could rename `action` to `secondaryAction` and then add `tertiaryAction` if `action` is already misleading though.

But it also might make more sense to handle this outside of `FlatList` entirely and instead just treat the extra stuff as content. Does material and/or material UI define how a third action should render in lists?

Reply:
They don't :( It's not "material" standard as far as material.io is concerned